av_downloader.py

```diff
@@ -48,7 +48,7 @@ def download(self, symbol, interval='1min'):
                 return ""
 
 if __name__ == '__main__':
-    dir_path = Path(Path.cwd(), 'av_historical_1m')
+    dir_path = Path('/av_historical_1m')
```

Comment:
Aight glad to see you figured it out, worked on my machine not sure why tho, maybe the directory was already there from a previous run and I did not notice